aspnetcore/fundamentals/metapackage.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Microsoft.AspNetCore.All metapackage for ASP.NET Core 2.X and later
+author: Rick-Anderson
+description: Microsoft.AspNetCore.All metapackage includes all supported packages.
+keywords: ASP.NET Core, NuGet, package, Microsoft.AspNetCore.All, metapackage
+ms.author: riande
+manager: wpickett
+ms.date: 07/16/2017
+ms.topic: article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: fundamentals/metapackage
+---
+
+#Microsoft.AspNetCore.All metapackage for ASP.NET Core 2.X
+
+This feature requires ASP.NET Core 2.X.
+
+The `Microsoft.AspNetCore.All` metapackage for ASP.NET Core includes all:
+
+* Supported packages by for ASP.NET Core.
+* Supported packages by for Entity Framework Core. 
+* Internal and 3rd-party dependencies used by ASP.NET Core and Entity Framework Core. 
+
+All the features of ASP.NET Core 2.X and Entity Framework Core 2.X are included in the [Microsoft.AspNetCore.All](https://www.nuget.org/packages/Microsoft.AspNetCore.All) package. The default templates use this package.
```

Comment:
~~The default templates use this package.~~
The default project templates use this package.